src/shell.ts

```diff
@@ -164,6 +174,8 @@ export class Shell {
         if (key == 'Escape') {
             if (form)
                 await this.escapeActiveForm()
+            else if (this.watchdog)
```

Comment:
you should let escapeWatchdog worry about that and just call it

Reply:
If as watchdog is ongoing, I don't want to call `map.showLog(false)` in the `else` statement so it's either this or I change the `else` to `else if (!this.watchdog)` which seems like a weird way to do it (if I'm calling `escapeWatchdog()` below anyway).

Reply:
Let me see if I get it, the user hits ESC and if there's a watchdog, it just stops it, otherwise it hides TWR. When using the normal shell CTRL-c doesn't make it go away, but exit does. We have the `hide` for that and I think it'll make more sense for ESC to not hide TWR.